ci/test.sh

```diff
@@ -87,4 +87,15 @@ pushd linuxdeploy-plugin-qt-examples/QtWidgetsApplication
         "$linuxdeploy_bin" --appdir "$PWD"/AppDir --plugin qt --output appimage || exit 1
         mv -v *AppImage "$build_dir" || exit 1
     popd
+
+    mkdir build-platforms
+    pushd build-platforms
+        export EXTRA_PLATFORM_PLUGINS="libqoffscreen.so;libqminimal.so"
+        qmake CONFIG+=release PREFIX=/usr ../QtWidgetsApplication.pro || exit 1
+        INSTALL_ROOT="$PWD"/AppDir make install || exit 1
+
+        "$LINUXDEPLOY_BIN" --appdir "$PWD"/AppDir --plugin qt --output appimage || exit 1
+        APPIMAGE=(*AppImage)
```

Comment:
I guess it makes sense to use `*.AppImage` as a pattern. But generally, I'd prefer you to set the output filename (see above).